src/ngraph/runtime/interpreter/int_executable.cpp

```diff
@@ -70,12 +70,22 @@ runtime::interpreter::INTExecutable::INTExecutable(const shared_ptr<Function>& f
 #else
     m_function = clone_function(*function);
 #endif
+    auto is_supported = [](const Node& node) {
+        bool retval = false;
+        switch (INTExecutable::get_typeid(node))
+        {
+        case OP_TYPEID::Squeeze:
+        case OP_TYPEID::Unsqueeze: retval = true; break;
+        default: break;
```

Comment:
Looks weird but this is what clang-format does.

Reply:
It must be right then. 😄